### PR TITLE
158/login/restrict-vm-access-for-non-root-users

### LINKED ISSUE #158: restriction for users without root rights
[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)

**User Story**

[//]: # (Be descriptive about your user story. Who requests a feature and what's his intent?)

As a user I want to only be able to start or stop vms if I have root rights to them (#74), so that only people with the right rights can send start and stop requests to a vm.

**Acceptance Criteria:**

- [ ] Story #74 is only available when a user hast root-rights on the vm.

[//]: # (Please don't forget to add the priority and complexity label to this issue.)

### PR DESCRIPTION
**Pull request by lassesteffen**
Fixes #158

### CODE REVIEW

File: spec/controllers/vms_controller_spec.rb [hunk 1]
```diff
@@ -74,43 +89,50 @@
...
+
+        it 'returns online VMs if requested' do
+          expect(subject.vms).to satisfy('include online VMs') { |vms| vms.any?(&:powered_on?) }
+        end
 
-      it 'returns offline VMs if requested' do
-        get :index, params: { down_vms: 'true' }
-        expect(subject.vms).to satisfy('include offline VMs') { |vms| vms.any?(&:powered_off?) }
-        expect(subject.vms).not_to satisfy('include online VMs') { |vms| vms.any? { |vm| !vm.powered_off? } }
+        it 'does not return not online vms' do
```
Comment (janwestphal306) [inline spec/controllers/vms_controller_spec.rb]: not online => offline

File: spec/controllers/vms_controller_spec.rb [hunk 2]
```diff
@@ -74,43 +89,50 @@
...
 
-    it 'returns http success' do
-      delete :destroy, params: { id: 'my insanely cool vm' }
-      expect(response).to have_http_status(:success)
-      skip
+        it 'returns offline VMs' do
+          expect(subject.vms).to satisfy('include offline VMs') { |vms| vms.any?(&:powered_off?) }
+        end
+
+        it 'does not return not offline vms' do
```
Comment (janwestphal306) [inline spec/controllers/vms_controller_spec.rb]: not offline => online

File: spec/views/vms/show.html.erb_spec.rb [hunk 3]
```diff
@@ -1,196 +1,217 @@
 # frozen_string_literal: true
 
-# TODO: klären, inwieweit View tests gemacht werden sollen
+# TODO: check, to which degree view tests should be created
```
Comment (janwestphal306) [inline spec/views/vms/show.html.erb_spec.rb]: nice change :D

Comment (lassesteffen) [reply spec/views/vms/show.html.erb_spec.rb]: Rubocop was unhappy

**Pull request merged**